system-addon/test/unit/lib/Store.test.js

```diff
@@ -38,55 +49,129 @@ describe("Store", () => {
     store.dispatch({type: "FOO"});
     assert.calledOnce(store._mm.middleware);
   });
-  describe("#init", () => {
-    it("should dispatch an init action", () => {
-      sandbox.spy(store, "dispatch");
+  describe("#initFeed", () => {
+    it("should add an instance of the feed to .feeds", () => {
+      class Foo {}
+      store._feedConstructors = {foo: () => new Foo()};
```

Comment:
Aren't these really factories rather than constructors?  Changing the name here would make the code a bit easier to read, I think.